lib/chitter.rb

```diff
@@ -0,0 +1,39 @@
+require 'pg'
+class Chitterpeep
+   
```

Comment:
Great to see that you've tried to add the timestamp. I know you've already mentioned that you couldn't display the time how you'd like, however, it is functioning well.